Kitodo/src/main/java/org/kitodo/production/services/data/RulesetService.java

```diff
@@ -222,18 +229,35 @@ private QueryBuilder getRulesetsForCurrentUserQuery() {
         return query;
     }
 
+    public static MonthDay getYearBegin(int processId)
+            throws ConfigurationException, DAOException, DataException, IOException, RulesetNotFoundException {
+        String baseType = ServiceManager.getProcessService().getBaseType(processId);
+        Process process = ServiceManager.getProcessService().getById(processId);
+        RulesetManagementInterface ruleset = openRuleset(process.getRuleset());
+        List<LanguageRange> priorityList = LanguageRange
+                .parse(ServiceManager.getUserService().getCurrentUser().getMetadataLanguage());
+        ruleset.getStructuralElementView(baseType, "", priorityList);
+        StructuralElementViewInterface newspaperView = ruleset.getStructuralElementView(baseType, "", priorityList);
+        StructuralElementViewInterface yearDivisionView = NewspaperProcessesGenerator.nextSubView(ruleset,
+            newspaperView, "");
+        DatesSimpleMetadataViewInterface yearSimpleMetadataView = yearDivisionView.getDatesSimpleMetadata()
+                .orElseThrow(ConfigurationException::new);
```

Comment:
Why are you introducing a exception class from an other package name space instead of using or creating a new class in kitodo package name space?